src/Script.js

```diff
@@ -31,15 +31,14 @@ import {AllServers, processSingleServerGrowth}  from "./Server";
 import {Settings}                               from "./Settings";
 import {post}                                   from "./ui/postToTerminal";
 import {TextFile}                               from "./TextFile";
-
 import {parse, Node}                            from "../utils/acorn";
 import {Page, routing}                          from "./ui/navigationTracking";
+import numeral                                  from "numeral/min/numeral.min";
```

Comment:
I'd be interested in the difference that bringing in the minified version has vs. bringing in the normal version and letting Webpack minify the entire bundle.